From ca952acd72964dca53a9f50cf728dd1d31d48117 Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Tue, 7 Feb 2017 17:52:00 +0100
Subject: [PATCH] mm: Re-export ioremap_page_range

We need this in Jailhouse to map at specific virtual addresses, at
least for the moment.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
(cherry picked from commit 94bb285491a9a9e15c82c0761505b1073d6b7a47)
---
 lib/ioremap.c | 1 +
 1 file changed, 1 insertion(+)

--- a/lib/ioremap.c
+++ b/lib/ioremap.c
@@ -231,3 +231,4 @@ int ioremap_page_range(unsigned long add
 
 	return err;
 }
+EXPORT_SYMBOL_GPL(ioremap_page_range);
